config/samples/capsule_v1alpha1_capsuleconfiguration.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
installing with `make deploy` but it seems the default configuration is not created

```
$ kubectl get capsuleconfigurations
No resources found in capsule-system namespace.
$ kubectl get capsuleconfigurations -A
No resources found
```

but it is referenced in the manager

```yaml
...
    Command:
      /manager
    Args:
      --enable-leader-election
      --zap-encoder=console
      --zap-log-level=debug
      --configuration-name=default
```

Reply:
Solved with commit 537e4a3